Source/SA/DPD/DPD.cs

```diff
@@ -0,0 +1,204 @@
+﻿using NationalInstruments.RFmx.InstrMX;
+using NationalInstruments.RFmx.SpecAnMX;
+using NationalInstruments.ModularInstruments.NIRfsg;
+using NationalInstruments.ModularInstruments.NIRfsgPlayback;
+using static NationalInstruments.ReferenceDesignLibraries.SG;
+using System;
+
+namespace NationalInstruments.ReferenceDesignLibraries.SA
+{
+    public static class DPD
+    {
+        #region Type_Definitionss
+        public struct CommonDPDConfiguration
+        {
+            public double dpdMeasurementInterval;
+            public RFmxSpecAnMXDpdMeasurementSampleRateMode dpdSampleRateMode;
+            public double sampleRate;
+            public RFmxSpecAnMXDpdSignalType signalType;
+            public RFmxSpecAnMXDpdReferenceWaveformIdleDurationPresent idleDurationPresent;   //TBD pull idle present flag from SG
```

Comment:
can't this be pulled from the SG Waveform struct

Reply:
@cindyxie This one still remains; please change this to use the waveform struct sample rate

Reply:
I've removed the idleduration from RFmxDPD. Can you double check?